src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -14,6 +14,7 @@
       "description": "The set of runs contained in this log file.",
       "type": "array",
       "minItems": 1,
+      "uniqueItems": true,
```

Comment:
Per our discussion, agreed. Run's can't be completely empty. They always have at least a tool name. So it's useful to say "I ran FxCop twice" (and there were no results).